pkg/distro/fedora/distro_internal_test.go

```diff
@@ -0,0 +1,28 @@
+package fedora
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/osbuild/images/pkg/blueprint"
+	"github.com/osbuild/images/pkg/disk"
+	"github.com/osbuild/images/pkg/distro"
+	"github.com/osbuild/images/pkg/distro/distro_test_common"
+)
+
+// math/rand is good enough in this case
+/* #nosec G404 */
+var rng = rand.New(rand.NewSource(0))
+
+func TestESP(t *testing.T) {
+	var distros []distro.Distro
+	for _, distroName := range []string{"fedora-40", "fedora-41", "fedora-42"} {
```

Comment:
We could extend this to `fedora-99` and not have to worry about keeping the list up-to-date.  Or maybe we can add a follow-up item to iterate over all known distro versions based on `test/data/repositories/`.

Reply:
I tried this, and it creates a hard-to-break import loop. Let's figure this in a separate PR. :)